data_structures/hash_table/hash_table.py

```diff
@@ -0,0 +1,81 @@
+from linked_list import LinkedList
+
+
+class HashTable:
+    def __init__(self, max_size=1024):
+        """
+        This initiatates the hash table with empty linked lists in each bucket.
+        """
+
+        if type(max_size) is not int:
+            raise TypeError
```

Comment:
It is best practice to add a description to your exceptions to inform the user what they have done wrong.